confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeConf.java

```diff
@@ -32,10 +33,12 @@ public class ConfigNodeConf {
 
   /** used for communication between data node and config node */
   private int rpcPort = 22277;
-
   /** used for communication between config node and config node */
   private int internalPort = 22278;
 
+  /** Thrift socket and connection timeout between data node and config node */
+  private int connectionTimeoutInMS = (int) TimeUnit.SECONDS.toMillis(20);
```

Comment:
This field should also be added to iotdb-confignode.properties and ConfigNodeDescriptor.

Reply:
Fixed